common/components/constants/LinkConstants.js

```diff
@@ -52,6 +57,11 @@ export const LinkDisplayConfigurationByUrl: $ReadOnlyArray<LinkSourceDisplayConf
     sourceDisplayName: "LinkedIn",
     iconClass: GlyphStyles.LinkedIn,
   },
+  {
+    sourceUrlPattern: new RegExp("Figma.com", "i"),
+    sourceDisplayName: "Figma",
+    iconClass: GlyphStyles.Figma,
+  },
```

Comment:
The only difference between this and the change at line 30 is capitalization, which should be covered by the "i" flag.